src/Compilers/CSharp/csc/Program.cs

```diff
@@ -24,7 +24,13 @@ public static int Main(string[] args)
         }
 
         private static int MainCore(string[] args)
-            => DesktopBuildClient.Run(args, RequestLanguage.CSharpCompile, Csc.Run, new DesktopAnalyzerAssemblyLoader());
+        {
+#if NET46
+            return DesktopBuildClient.Run(args, RequestLanguage.CSharpCompile, Csc.Run, new DesktopAnalyzerAssemblyLoader());
+#else
+            return CoreClrBuildClient.Run(args, RequestLanguage.CSharpCompile, Csc.Run, new CoreClrAnalyzerAssemblyLoader());
+#endif
```

Comment:
I think we should be able to just have a portable BuildClient here, instead of multitargeting. (I wanted to preserve code/behavior of CscCore as much as possible here, so I kept the original implementation, but if that's something that's okay, I'll merge them)

Reply:
i agree it could be portable but i think we should do that in a follow up pr